packages/prisma/src/prisma/schema.prisma

```diff
@@ -239,15 +239,15 @@ model Session {
   finishedAt  DateTime?
 
   accessMode AccessMode    @default(PUBLIC)
-  status     SessionStatus
+  status     SessionStatus @default(PREPARED)
 
   activeBlock Int            @default(-1)
   blocks      SessionBlock[]
 
-  owner   User   @relation(fields: [ownerId], references: [id])
+  owner   User   @relation(fields: [ownerId], references: [id], onDelete: Cascade, onUpdate: Cascade)
   ownerId String @db.Uuid
 
-  course   Course? @relation(fields: [courseId], references: [id])
+  course   Course? @relation(fields: [courseId], references: [id], onDelete: SetNull, onUpdate: Cascade)
```

Comment:
Sessions can exist without courses, so if a course is deleted, its sessions and results could be kept around (but maybe not best user experience).